src/app/common/stores/applications/SsvWeb/Operator.store.ts

```diff
@@ -207,14 +208,21 @@ class OperatorStore extends BaseStore {
   }
 
   /**
-   * Check if operator registrable
+   * Get max validators count
    */
-  async isOperatorRegistrable(validatorsRegisteredCount: number) {
+  async getMaxValidators() {
     const walletStore: WalletStore = this.getStore('Wallet');
     const contract: Contract = walletStore.getterContract;
     if (this.operatorValidatorsLimit === 0) {
       await contract.methods.getValidatorsPerOperatorLimit().call().then((maxValidators: number) => this.operatorValidatorsLimit = maxValidators);
```

Comment:
It is better to make assignment on a top level: `this.operatorValidatorsLimit = await contract....`
rather than doing it in a callback, as it doesn't create nested references which is always better

Reply:
+